vectorfunctions.hh

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include<vector>
+
+//elementwise addition, if vectors are of the same length
+template<class numbertype=double> void operator+=(std::vector<numbertype> &v1, std::vector<numbertype> v2) {
+  if(v1.size() != v2.size()){
+      std::cerr << "vectors do not match in size! No addition took place." << std::endl;
+      return;
+  }
+  for(size_t i = 0; i < v1.size(); i++){
```

Comment:
Maybe this can be improved as explained here:
https://riptutorial.com/openmp/example/23425/addition-of-two-vectors-using-openmp-parallel-for-construct

Reply:
In general this is a good idea, but at the moment I'm only using it to add vectors with two elements. I think in this case it would add more overhead than savings to parallelize this.